Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -454,6 +454,7 @@ public static void RegisterRoutes(IKernel kernel, RouteCollection routes)
             // Jobs
             routes.MapHttpWebJobsRoute("list-all-jobs", "", "", new { controller = "Jobs", action = "ListAllJobs" }, new { verb = new HttpMethodConstraint("GET") });
             routes.MapHttpWebJobsRoute("list-triggered-jobs", "triggered", "", new { controller = "Jobs", action = "ListTriggeredJobs" }, new { verb = new HttpMethodConstraint("GET") });
+            routes.MapHttpWebJobsRoute("list-triggered-jobs-swagger", "triggered", "/swagger", new { controller = "Jobs", action = "ListTriggeredJobsInSwaggerFormat" }, new { verb = new HttpMethodConstraint("GET") });
```

Comment:
this is going to be specified in the nuget package for webjobs as a service, where you can not add a query string. 

we discussed the problem of having "swagger" as not being available as a webjob name and i think it was ok 

the usual path for the swagger endpoint is api/swagger/v1 for example but it did not make sense for triggered web jobs to be api/triggeredwebjobs/api/swagger/v1